docker-compose.yml

```diff
@@ -32,8 +54,10 @@ services:
       driver: json-file
       options:
         max-size: "10m"
-
+    healthcheck:
```

Comment:
The healthcheck allows us to chain the containers together. We can wait for mariadb to get running